### DISCUSSION TITLE
fine tuning whisper model to onnx+

### DISCUSSION THREAD

**Discussion by bobur-amirov** [k2-fsa/sherpa-onnx#467] [Q&A]
how do I convert my fine-tuned model to onnx? export-onnx.py is only working for whisper itself models!

**Comment by csukuangfj** [comment]
Is your code open-sourced? If yes, could you share the link to your code?

If not, could you tell us what changes you have made to the original whisper model?

**Reply by bobur-amirov** [reply]
my code not open-sourced.

https://github.com/huggingface/transformers/tree/main/examples/pytorch/speech-recognition I have configured  with my dataset following this tutorial.

**Reply by csukuangfj** [reply]
I suggest that you follow export-onnx.py to export your own model to onnx.
